distribution/pom.xml

```diff
@@ -44,6 +49,7 @@
         <module>install-profiles</module>
         <module>ddf</module>
         <module>test</module>
+        <module>docs</module>
```

Comment:
Not that I recall. I know it was originally last when I first created it, but it doesn't need to be.